98_exercises.Rmd

```diff
@@ -291,24 +291,87 @@ See the OMA book for examples.
 
 ### MultiAssayExperiment (MAE) data container
 
-1. Create TreeSE data containers from individual CSV files.
-2. Combine TreeSE into MAE.
-3. Check that each individual experiment of MAE equals corresponding TreeSE.
-4. Take a subset of MAE (e.g., 10 first samples), and observe the subsetted MAE.
-
-Useful functions: DataFrame, TreeSummarizedExperiment, matrix, rownames, colnames, MultiAssayExperiment, ExperimentList, SimpleList
-
+The following exercise walks you through disassembling a MAE object in
+order to store it as multiple separate csv files.
+
+1. Import the microbiomeDataSets package
+2. Store the HintikkaXOData MAE into a variable named `mae`
+3. Which and how many experiments are contained by the MAE?
+4. Split the MAE into TreeSEs named `microbiota`, `metabolites` and `biomarkers`
+5. Which and how many assays are contained by each TreeSE?
+6. Store the assays of each TreeSE into a variable named `<TreeSE name>_assays`
+7. Write a csv file for each assays object with the `write.csv` function
```

Comment:
doing this for one assay could be enough for exercise. They can do the rest if they want to but I would not make it a too repetitive example.

Reply:
In fact I am not sure if we need to have exercise on how to store CSV files here in this context. It is basic R skill, not MAE specific skill. We could also just give web location for the CSV files and suggest to try TreeSE -> MAE construction based on those directly